.github/actions/helm-default/action.yaml

```diff
@@ -29,7 +29,13 @@ runs:
         echo sha=${SHA} >> $GITHUB_OUTPUT
 
         CILIUM_INSTALL_DEFAULTS="--chart-directory=${{ inputs.chart-dir }} \
-          --helm-set=image.repository=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/cilium-ci \
+          --helm-set=debug.enabled=true \
+          --helm-set=bpf.monitorAggregation=none"
+
+        # only add SHA to the image tags if it was set
+        if [ -n "${SHA}" ]; then
+          echo sha=${SHA} >> $GITHUB_OUTPUT
```

Comment:
Looks like this is redundant, isn't it?

Reply:
oh nice catch. I forgot to delete line 29